tools/bot-repo-cleanup/cleanup.py

```diff
@@ -0,0 +1,26 @@
+#!venv/bin/python3
+
+# Obtained with `pip install PyGithub`, better within a venv
+from github import Github
+from github.Workflow import Workflow
+
+# API token for yunohost-bot, with "delete_repo" right
+g = Github("TOKEN_REPLACE_ME")
```

Comment:
```suggestion
# API token for yunohost-bot, with "delete_repo" right
g = Github(open(".github_token").read().strip())
```

Might be better to have the token stored in another file to prevent mistakenly committing a token later ? :sweat_smile: